### PR TITLE
Moving `graphql-ws` to a dynamic import and display a nice message when not found

### LINKED ISSUE #2405: Can't resolve `graphql-ws` from `@graphiql/toolkit/esm/create-fetcher`
Seems like the recent release of `@graphiql/toolkit` has broken some of our tests. Maybe due to a missing dependency?

```
Module not found: Error: Can't resolve 'graphql-ws' in '/tmp/backstage-e2e-Z2eQgF/test-app/node_modules/@graphiql/toolkit/esm/create-fetcher'

error Command failed with exit code 1.
```

From https://github.com/backstage/backstage/runs/6420680730?check_suite_focus=true

### PR DESCRIPTION
**Pull request by benjdlambert**
Fixes #2405

### CODE REVIEW

File: packages/graphiql-toolkit/src/create-fetcher/lib.ts [hunk 1]
```diff
@@ -76,13 +80,24 @@ export const createWebsocketsFetcherFromUrl = (
...
     wsClient = createClient({
       url,
       connectionParams,
     });
     return createWebsocketsFetcherFromClient(wsClient);
   } catch (err) {
+    if (errorHasCode(err)) {
+      if (err.code === 'MODULE_NOT_FOUND') {
+        throw Error(
+          'You need to install the `graphql-ws` package to use websockets',
```
Comment (thomasheyenbrock) [inline packages/graphiql-toolkit/src/create-fetcher/lib.ts:97]: We could be more specific here to educate where this error is coming from. (We don't expose the `createWebsocketsFetcherFromUrl` function publicly, you end up in this code path when passing a `subscriptionUrl` to `createGraphiQLFetcher`.)
```suggestion
          'You need to install the `graphql-ws` package to use websockets when passing a `subscriptionUrl`',
```

**Review by thomasheyenbrock — COMMENTED** [review]
Awesome, thanks for jumping on this ❤️ Just a tiny comment around the error message

**Comment by benjdlambert** [comment]
Updated 👍 @thomasheyenbrock

**Review by thomasheyenbrock — COMMENTED** [review]
@benjdlambert seems like the new test is not passing yet 👀 can you have another look please?

**Comment by benjdlambert** [comment]
Derp. Should be good now!

**Pull request merged**